src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -479,7 +479,7 @@ int equal_cmp(void* /*p1*/, void* /*p2*/) {
   // token or client stats.  A better approach might be to find somewhere
   // other than the subchannel args to store the LB token and client
   // stats.  They could be stored in a map and then looked up for each
-  // call (although we'd need to make sure our Map<> implementation is
+  // call (although we'd need to make sure our std::map<> implementation is
```

Comment:
I think the clause in parens can just go away now.

Reply:
Done.